fedmsg_atomic_composer/composer.py

```diff
@@ -59,6 +59,8 @@ def compose(self, release):
             else:
                 traceback.print_exc()
             release['result'] = 'failed'
+        finally:
+            self.cleanup_logs()
```

Comment:
Yes, turns out renaming a function after writing it first time doesn't always go right :D.
Fixed.